lib/ansible/module_utils/k8s/raw.py

```diff
@@ -60,24 +60,30 @@ def __init__(self, *args, **kwargs):
                 'apiVersion': api_version,
                 'metadata': {
                     'name': name,
-                    'namespace': namespace
+                    'namespace': namespace,
                 }
             }]
 
     def execute_module(self):
         changed = False
+        facts = False
```

Comment:
Do you still need any of the changes to this file? Looks like they're no longer used.

Reply:
It, or some alternative implementation, is still needed, as `List` still gets stripped off `kind` before the call to `perform_action` (which means that the bug where passing `List` causes a patch to be attempted still occurs because it doesn't know not to attempt a patch). The alternative is not to strip `List` off `kind` - really you're the best person to answer that as the code is authored by you.

I can completely strip out these changes and leave the original bug, now that I can use `k8s_facts` rather than `k8s` with `kind`=`*List` - depends what you want to do about the bug.

Reply:
I've removed the deprecation though.